src/components/explore/QuantileExplorerView.svelte

```diff
@@ -129,13 +129,13 @@ $: selectedData = filterQuantileData(data, currentAggregation, currentKey);
               showViolins={true}
               aggregationLevel={aggregationLevel}
               binColorMap={percentileLineColorMap}
-              pointMetricType={probeType === 'histogram' ? 'transformedPercentiles' : 'percentiles'}
-              overTimePointMetricType={probeType === 'histogram' ? 'transformedPercentiles' : 'percentiles'}
+              pointMetricType={probeType === 'log' ? 'transformedPercentiles' : 'percentiles'}
```

Comment:
moving these string from `histogram` to `log` / `linear` (for scalars) will prepare us for Glean products.